Chapter19/19-5/my_db_gen_tests.erl

```diff
@@ -0,0 +1,131 @@
+-module(my_db_gen_tests).
+-import(my_db_gen, [start_link/0, stop/0, write/2, delete/1, read/1, match/1]).
+
+% -define(NOTEST, true).
+-include_lib("eunit/include/eunit.hrl").
+
+
+
+%% code:add_path("my_db-3.0/ebin/").
+
+
+
+start_link_test_() ->
```

Comment:
I suggest to no start and stop application on each test case, or if it's necessary use `foreach` fixture.

Reply:
Combined all the tests in to one so the server only starts and stops once